src/compiler/checker.ts

```diff
@@ -1473,8 +1476,6 @@ namespace ts {
             return undefined;
         }
 
-        // This is for caching the result of getSymbolDisplayBuilder. Do not access directly.
-        let _displayBuilder: SymbolDisplayBuilder;

```

Comment:
This was reported by a linter error? I think the original location was more appropriate.